src/shared/layouts/components/sessionTimer.tsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import axios from 'axios';
 import { Helmet } from 'react-helmet-async';
 import Modal from 'react-modal';
-import { Context } from '../../contexts/application';
+import { ApplicationState, Context } from '../../contexts/application';
+import { hasClientError } from '../../helpers/status-helpers';
 
 const getDefaultExpiryDate = (defaultTimeoutSeconds: number) => new Date().getTime() + defaultTimeoutSeconds * 1000;
 const getRemainingSeconds = (targetDate: number) => Math.floor((targetDate - new Date().getTime()) / 1000);
+
 const keepAlive = () => axios.get('/api/keepalive')
     // eslint-disable-next-line no-undef
-    .catch(() => window.location.reload());
+    .catch((error) => {
+        setTimeout(() => window.location.reload(), 1000); // wait one second before refreshing
```

Comment:
Why the one second wait?

Reply:
If /api/keepalive returns an error, the catch block refreshes the pages, causing keepalive() to be run again. I thought it would be more efficient to wait a second to allow any potential errors to resolve, rather than trying again immediately.

Reply:
If there's an error we refresh, just not sure what the wait will achieve exactly. Also keeping in mind we're using the `keepAlive()` function in other places and we want to keep that the same.

Reply:
I've removed this.